### ISSUE TITLE
day-mode should be extended to editor

### ISSUE THREAD

**Issue by igalic** [hedgedoc/hedgedoc#763]
**Which part of the project should be enhanced?**

Currently, when toggling between day and night mode, only the toolbar, and preview are changed.
I would like for the actual editor pane to be changed, too.

**Is your enhancement request related to a problem? Please describe.**

On a bright and sunny day, the dark editor is really really hard to use.

**Describe the solution you'd like**

The sun and moon toggle the editor from bright to dark, not just the toolbar

**Describe alternatives you've considered**

alternatively, if people find this offensive to their tastes, i can imagine having different sets of editor styles on the bottom to choose from.

Although i think this solution would be less intuitive and less accessible.

**Additional context**

I am "old" and I'm only getting older, and my eyes are only gonna get worse.

**Comment by DerMolly** [comment]
Yes this is the behavior in 1.x. There is also the possibility to switch the editor style.

![image](https://user-images.githubusercontent.com/3978662/105722895-6dc5af00-5f26-11eb-9bc0-8aa5b4effeaa.png)

I'm not sure there are further plans to change this behavior for 2.0. 

Also I thought there was already an issue about supporting multiple themes for codemirror (that's the lib that does most of the editor part in HedgeDoc) in the [react-client repo](https://github.com/hedgedoc/react-client/issues), but apparently I'm mistaken.

**Comment by igalic** [comment]
that is really hard to see or hit, on mobile
so perhaps just following, by default, the OS / browser's dark / bright preference would be preferable from a usability standpoint.

aaaaand, given that this repo is now only the backend, maybe it would make sense to migrate this issue to https://github.com/hedgedoc/react-client/issues
